Source/WebCore/css/StyleSheetContents.cpp

```diff
@@ -504,7 +504,7 @@ Document* StyleSheetContents::singleOwnerDocument() const
 
 static bool traverseRulesInVector(const Vector<Ref<StyleRuleBase>>& rules, NOESCAPE const Function<bool(const StyleRuleBase&)>& handler)
 {
-    for (auto& rule : rules) {
+    for (const auto& rule : rules) {
```

Comment:
Why is this change necessary?

Reply:
We don't do const variables in WebKit

Reply:
It's not necessary, just marking the ref as const for more security